### DISCUSSION TITLE
Trouble getting started

### DISCUSSION THREAD

**Discussion by trevor-powdrell** [lorserker/ben#148] [General]
I am having trouble getting started. When running ./conda_setup.sh I get the following errors

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
grpcio-tools 1.68.0 requires protobuf<6.0dev,>=5.26.1, but you have protobuf 4.25.5 which is incompatible.

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
tensorflow 2.13.0 requires protobuf!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.20.3, but you have protobuf 5.29.0 which is incompatible.

**Comment by ThorvaldAagaard** [comment]
I recommend using Tensorflow 2.18, but the install scripts might not reflect that.
What happens if you just ignore the errors and try to start BEN?

The simplest way to check is to switch to the src-directory and run
```
python game.py
```
That will bid and play a random board.

If it fails, then send the output

**Comment by trevor-powdrell** [comment]
Just ignoring the error and trying python game.py just complained about missing keras from TensorFlow. So I used the conda_setup_tf2.sh that uses Tensorflow 2.18 and it all worked. Thanks.

**Reply by ThorvaldAagaard** [reply]
Great, I am trying to maintasin the TF1.X version developed by Lorand, but perhaps I better delete it, as I normally just focus on the latest version of Tensorflow.
Let me know how it goes.
